mteb/models/promptriever_models.py

```diff
@@ -55,6 +55,15 @@ def loader_inner(**kwargs: Any) -> Encoder:
     open_source=True,
     revision="01c7f73d771dfac7d292323805ebc428287df4f9-30b14e3813c0fa45facfd01a594580c3fe5ecf23",  # base-peft revision
     release_date="2024-09-15",
+    n_parameters=7_000_000,
+    memory_usage=None,
+    max_tokens=None,
+    embed_dim=None,
+    license="apache-2.0",
+    reference="https://huggingface.co/samaya-ai/promptriever-llama2-7b-v1",
+    similarity_fn_name="cosine",
+    framework=["PyTorch"],
```

Comment:
Is Tevatron (though also PyTorch).  

Max tokens is 4096 and embed dim 4096. 

This is true for all Promptriever models (except the llama3 one which is 8k).

Reply:
added